modules/ROOT/pages/overview.adoc

```diff
@@ -53,5 +51,8 @@ Multi-tenancy is not supported.
 
 |===
 
-+++*+++ There are known limitations with those runtime versions: 
+*Bonita Central* will be able to connect and display information in : 
+
+- Degraded mode with Bonita Runtime from 2021.1 till Bonita Runtime 2022.2 based on their technical capabilities 
```

Comment:
I thought that Bonita will be not supported anymore when Bonita Central will be released along with Bonita 2023.1.
Is it not the case anymore?